recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+if [[ "${target_platform}" == "osx-64" ]]; then
+    export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake . -B ./build -G Ninja ${CMAKE_ARGS} -DSEAL_BUILD_DEPS=OFF -DSEAL_BUILD_TESTS=ON
+cmake --build ./build -j ${CPU_COUNT}
```

Comment:
this diverges from the win version too (no `--config Release`)

Reply:
well, this is also a minor nit, as the default is Release (presumably on both platforms)

Reply:
`CMAKE_ARGS` contains `-DCMAKE_BUILD_TYPE=Release`a lready. Also, `--config` is for multi-config generators ("Visual Studio", "Xcode" and "Ninja Multi-Config"). `CMAKE_ARGS` on Windows doesn't seems to set [CMAKE_CONFIGURATION_TYPES](https://cmake.org/cmake/help/latest/variable/CMAKE_CONFIGURATION_TYPES.html#variable:CMAKE_CONFIGURATION_TYPES) which means that it will generate all types and default to Debug.

Though, since I use ninja now on all platforms (which wasn't the case initially), I can get rid of `--config` on the Windows side and rely on `CMAKE_ARGS` to set the build type to Release.